robottelo/upgrade_utility.py

```diff
@@ -0,0 +1,150 @@
+"""Common Upgrade test utilities """
+
+from nailgun import entities
+from wait_for import wait_for
+from fabric.api import execute, run
+from robottelo.test import settings
+from upgrade.helpers.docker import docker_execute_command
+from robottelo.api.utils import call_entity_method_with_timeout
+
+
+class CommonUpgradeUtility(object):
+    """ CommonUpgradeUtility class for upgrade test cases """
+
+    def __init__(self, container_id=None, package=None):
+        self.client_container_id = container_id
+        self.package = package
```

Comment:
I think you can remove the package from here and add it in the method, package is not constant over the class whereas container_id will be constant most of the time over a test case.